app/lib/katello/validators/smart_proxy_default_capsule_validator.rb

```diff
@@ -0,0 +1,9 @@
+module Katello
+  module Validators
+    class SmartProxyDefaultCapsuleValidator < ActiveModel::Validator
+      def validate(record)
+        record.errors[:base] << _("Only one default capsule is allowed") if (record.has_feature?('Pulp') && SmartProxy.pulp_master && SmartProxy.pulp_master != record)
```

Comment:
Should `record.pulp_master?` be used instead of  `record.has_feature?('Pulp')` ?

Reply:
@sjha4 watcha think?

Reply:
@jturel: So this is a little complicated to do on the katello side alone and this fix only stops updating a second pulp master. We need one small change on foreman side to make it easier to stop creation..I will open a PR in foreman and change this accordingly..
Will mark WIP for now and get back to this..

Reply:
Sounds good!